web/modules/custom/collection/config/install/og.og_role.rdf_entity-collection-administrator.yml

```diff
@@ -1,6 +1,8 @@
 langcode: en
 status: true
-dependencies: {  }
+dependencies:
+  config:
+    - rdf_entity.rdfentity.collection
```

Comment:
Thanks for taking the time to export these dependencies, they have been required for a long time but we never got round to it.